### ISSUE TITLE
Weird behaviour

### ISSUE THREAD

**Issue by mindrunner** [Askannz/optimus-manager#127]
**Describe the bug**
I just installed optimus-manager on my thinkpad w530. However, the systray application seems quite unfunctional. I am trying to set bbswitch as the graphics switcher, but it always defaults to nouveau. However, nouveau is not loaded and blacklisted. I cannot change any settings.
How can we figure out whats going wrong?

**System info**
Please include :
- Your distribution: Arch
- Your desktop manager (KDE, GNOME, Cinnamon...): awesome-wm
- You display manager (SDDM, GDM, LightDM...): lightdm
- Your laptop model, if you think it could be related to the problem: Thinkpad W530
- The version of optimus-manager you are using : latest stable release (optimus-manager), or the latest Git version (optimus-manager-git): stable

**Logs**
```
Cannot setup GPU : kernel setup error : Cannot load nouveau : Failed to execute 'modprobe nouveau modeset=1' : modprobe: ERROR: could not find module by name='off'
modprobe: ERROR: could not insert 'off': Unknown symbol in module, or unknown parameter (see dmesg)
```

Can I use nouveau and NVIDIA at the same time? That does not make any sense? 
How can I use this with bbswitch and NVIDIA?

**Comment by Askannz** [comment]
The systray app is not part of optimus-manager, it's another project I have no control on. Please report any bug with it to https://github.com/Shatur95/optimus-manager-qt.

> Can I use nouveau and NVIDIA at the same time? That does not make any sense?
When you set `switching=nouveau`, optimus-manager will load the `nouveau` module as the driver for the Nvidia card in Intel mode. When you switch to Nvidia mode, `nouveau` is unloaded and `nvidia` is loaded instead.

**Comment by mindrunner** [comment]
Cool, thanks. I wasn't aware of this. 

I figured out the whole thing for me now. Maybe for future reference:

- My nouveau driver was blacklisted
- I used the default setting of nouveau as a swithcher
- Since, I had no config in my user-home, I could not change any settings via GUI 
- It seemed like it kinda worked, but not really....

---- > Switched to bbswitch, everything is working now (i am using NVIDIA driver)

Also no need to mention, that this has always been pain in the ass. I own a thinkpad w530 and can select between integrated, nvidia or optimus in BIOS. However, I realized, that the bbswitch/optimus-manager is able to spin up the NVIDIA card, even though I set intel-only in BIOS. (Followed by some unpredictable behavior)

Anyways, that was a bit off-topic. Maybe someone is interested. I am happy with my setup now.
